catalog/templates/column.html

```diff
@@ -48,12 +49,34 @@
 
    .. tab:: {"tab_name": "Boundaries by identifier in another table"}
 
+      Update the boundaries in another table matching the table's ID column.
+
+      .. code-block:: postgresql
+
+          WITH data AS (
+            SELECT cartodb_id AS id
+            FROM <table_to_update>
+          ),
+          meta AS (
+            SELECT OBS_GetMeta(ST_MakeEnvelope(-179, 89, 179, -89, 4326),
+              '[{"geom_id": "{{ col.id }}"}]'
+            ) meta
+          )
+          UPDATE <table_to_update> SET
+                geomref = data->0->>'geomref',
+                the_geom = (data->0->>'value')::Geometry,
+                the_geom_webmercator = ST_Transform((data->0->>'value')::Geometry, 3857)
+          FROM OBS_GetData(
+                (SELECT Array_agg(id) FROM data),
+                (SELECT meta FROM meta)) obs_data
+          WHERE <table_to_update>.cartodb_id = obs_data.id
```

Comment:
This query doesn't work at all in my account. I think it is related to the other comment about ids, but there are more things involved.

I think we should pass the geoids to this functions, and it can add the geometries (geocoding) to the original table. It seems this query is also trying to update the geomref, which doesn't make sense if that's the one being passed into the function.

Anyway, I feel like I may be missing something here 🤔

Reply:
The problem was in the id column name. It must be `geo_id` instead of `cartodb_id` as you mentiones in the other comment